pygfx/controls/_orbit.py

```diff
@@ -175,3 +175,43 @@ def update_camera(self, camera: "Camera") -> "OrbitControls":
         camera.position.copy(pos)
         camera.zoom = zoom
         return self
+
+    def add_default_event_handlers(self, canvas, camera):
+        """Apply the default interaction mechanism to a wgpu autogui canvas."""
+        canvas.add_event_handler(
+            lambda event: self.handle_event(event, canvas, camera),
+            "pointer_down",
+            "pointer_move",
+            "pointer_up",
+            "wheel",
+        )
+
+    def handle_event(self, event, canvas, camera):
+        """Implements a default interaction mode that consumes wgpu autogui events
+        (compatible with the jupyter_rfb event specification).
+        """
+        type = event["event_type"]
+        if type == "pointer_down":
+            xy = event["x"], event["y"]
+            if event["button"] == 1:
```

Comment:
This used to be the other way around; left mouse button is rotate, right mouse button is pan.

That also aligns with Three.js: https://threejs.org/examples/?q=orbit#misc_controls_orbit

Reply:
Fixed!